requirements/test-requirements.txt

```diff
@@ -34,3 +34,5 @@ shap
 evaluate
 nltk
 rouge_score
+# Required for FastAPI test app for Gateway
+httpx
```

Comment:
We can add this in `gateway.yml`

Reply:
`requirements/test-requirements.txt` contains too many packages. Installing all of them takes long.